MovieQuiz/Services/QuestionFactoryDelegate.swift

```diff
@@ -9,4 +9,7 @@ import Foundation
 
 protocol QuestionFactoryDelegate: AnyObject {              
     func didReceiveNextQuestion(question: QuizQuestion?)
+    func didLoadDataFromServer() // сообщение об успешной загрузке
+    func didFailToLoadData(with error: Error) // сообщение об ошибке загрузки
+    func didFailToLoadImage(with error: String) // сообщение об ошибке загрузки
```

Comment:
комментарии // - используются для пояснений в коде. Хорошо, когда комментарии полезны, иначе они просто захламляют код. 

В данном случае названия функций достаточно логичны, не вызывают вопросов, поэтому комментарии их лишь дублируют. Такие комментарии стоит удалить)

Но их можно оставить, в качестве док-комментариев, просто поменяв их расположение и объявление через ///:
```suggestion
    /// сообщение об успешной загрузке
    func didLoadDataFromServer() 
    /// сообщение об ошибке загрузки
    func didFailToLoadData(with error: Error) 
    /// сообщение об ошибке загрузки
    func didFailToLoadImage(with error: String) 
```

такие комментарии Xcode умеет интерпретировать и подсвечивать при нажатии на метод с зажатым option